prboom2/src/gl_main.c

```diff
@@ -1083,6 +1084,8 @@ void gld_EndDrawScene(void)
     // elim - Set the scissor back to the full viewport so post-scene draws can happen (ie StatusBar)
     dsda_GLSetRenderViewportScissor();
 
+    glsl_SetActiveShader(NULL);
```

Comment:
What do you think about migrating to a shader stack before doing this PR? There have been some bugs caused by nested logic and it might be good to clean that up before adding more.